ansible/recover_server.py

```diff
@@ -193,13 +193,15 @@ def _print_summary(jobs):
             if jobs[0].failed_task is not None:
                 output.append('Server %s cleanup failed, skip recovery.' % server)
             jobs = jobs[1:]
-        # start-vms output
-        if jobs[0].failed_task is None:
-            start_vms_result = 'Succeed.'
-        else:
-            start_vms_result = 'Failed.'
-        output.append('Server %s start-vms result: %s ' % (server, start_vms_result))
-        jobs = jobs[1:]
+
+        if vmtype != 'ceos':
```

Comment:
Don't we need to start vms when vmtype is 'vsonic'?

Reply:
Yes, we need to start vms when vmtype is `vsonic`. We only skip start vms when vmtype is `ceos.